source/_integrations/gentex_homelink.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: HomeLink
+description: Instructions on how to setup your HomeLink enabled car with Home Assistant.
+ha_category:
+  - Binary sensor
+ha_release: 2025.3
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@niaexa'
+  - '@ryanjones-gentex'
+ha_domain: gentex_homelink
+ha_platforms:
+  - binary_sensor
+ha_integration_type: integration
+ha_quality_scale: bronze
+---
+
+The HomeLink app allows [HomeLink](https://homelink.com) owners to trigger smart home routines in a variety of cloud-based home automation platforms from the comfort of your vehicle. When you arrive home, one HomeLink button press can adjust your thermostat, turn on your lights, disarm the security system, unlock the door, and begin playing your favorite music.
+
+Download the HomeLink app by scanning this QR code or following links to our app store pages:
+
+![HomeLink app QR code](/images/integrations/gentex_homelink/homelink-qr-code.png)
+
+- [HomeLink - Apple App Store](https://apps.apple.com/us/app/homelink/id1309231139)
+- [HomeLink - Google Play Store](https://play.google.com/store/apps/details?id=com.gentex.homelink.connect&hl=en-US&pli=1)
+
+## Prerequisites
+
+HomeLink is only compatible with HomeLink devices accessible via the HomeLink App. To prepare for an integration with Home Assistant:
+
+1. Install the HomeLink app through an app store of your choice.
+2. Create an account or sign in.
+3. Connect to your HomeLink mirror.
+
+## Configuration
+
+Upon enabling the HomeLink integration, you will be prompted to enter the email and password for your HomeLink account.
```

Comment:
```suggestion
{% configuration_basic %}
Email:
    description: "The email you used to log in to your HomeLink account."
Password:
    description: "The password to your HomeLink account."
{% endconfiguration_basic %}
```
You can use the [configration_basic block](https://developers.home-assistant.io/docs/documenting/integration-docs-examples#configuration_basic-block) to describe the information entered during setup. Please rephrase if I got the text wrong in the suggestion.